.github/workflows/ci.yml

```diff
@@ -32,6 +32,11 @@ jobs:
             - name: Test with oldest-supported-numpy
               run: uv run --no-dev --with oldest-supported-numpy python -m unittest discover tests
 
+            - name: Lint
+              run: |
+                  uv run ruff check --fix --exit-non-zero-on-fix
+                  uv run ruff format --check
```

Comment:
On phone so can't check myself - is there a Makefile we could use already? But don't create one if we don't already have it yet

Reply:
There is a makefile. I'll add this.

Reply:
https://github.com/tpgillam/mt2/pull/80/commits/a63d350cb9edd242fe0c683d878e303023674092

I did not just invoke `make lint`, because this action should fail fast on change, whereas the make rule should autoformat and fix without complaining.